doc/userguide/rules/http-keywords.rst

```diff
@@ -787,3 +787,4 @@ Notes
    pattern '<html' is absent from the first inspected chunk.
 
 -  ``file_data`` can also be used with SMTP
```

Comment:
A bit curious about why this wasn't removed here, but was for the other two Suri versions. Other than that, looks good to me!